roles/openshift_logging/handlers/main.yml

```diff
@@ -3,3 +3,14 @@
   systemd: name={{ openshift.common.service_type }}-master state=restarted
   when: (openshift.master.ha is not defined or not openshift.master.ha | bool) and (not (master_service_status_changed | default(false) | bool))
   notify: Verify API Server
+
+- name: Verify API Server
+  uri:
+    url: "{{ openshift.master.api_url }}/healthz/ready"
+    return_content: yes
+    status_code: 200
+  register: api_available_output
+  until: "'ok' in api_available_output.content"
```

Comment:
@ewolinetz is there a case where the /healthz/ready endpoint would return 200, but not return ok in the content?

I think you could update the until condition to be: `until: api_available_output | succeeded`